### PR TITLE
status: 2023q1: drm-drivers: corrections, suggestions

### PR DESCRIPTION
**Pull request by grahamperrin**
One correction to spelling, add a missing word, minor adjustments to grammar.

Whilst here: a handful of other suggestions.

### CODE REVIEW

File: website/content/en/status/report-2023-01-2023-03/drm-drivers.adoc [hunk 1]
```diff
@@ -16,8 +16,8 @@ Therefore this status report entry will cover more than just the last quarter.
 
 ==== Update to Linux 5.15 LTS and Linux 5.16
 
-As of this status report, the package:graphics/drm-kmod[] meta port still installs the DRM drivers from Linux 5.10 (released on December 13, 2020).
-This version lacks support for recent GPUs, in particular Intel 12th gen Alder Lake ones.
+As of this status report, the package:graphics/drm-kmod[] meta port still installs the DRM drivers from Linux 5.10 (released on December 13, 2020) on FreeBSD 13.1 and greater.
```
Comment (grahamperrin) [inline website/content/en/status/report-2023-01-2023-03/drm-drivers.adoc:19]: @evadot @dumbbell would you like to mention 13.1 (or `1301000`) here, as suggested? 

For less knowledgeable readers to _not_ wrongly assume that `drm-510-kmod` is (also) a default on 12.4.

**Review by lsalvadore — APPROVED** [review]
Approved by the status team, once you have decided if mentioning 13.1 or not.

**Review by dumbbell — APPROVED** [review]

**Comment by lsalvadore** [comment]
@grahamperrin: I am fine with you waiting to push this as long as it does not slow down the status reports publication. For now, as you know, we are still waiting some reports and working on some submissions, so there is no hurry.

However, please let me encourage you to play a bit with git, in particular with `git branch`, `git cherry-pick` and `git reset`. For example, in this case, you can push `af41f05cfc` without pushing `d4f7257766` by doing the following:

- create a duplicate of your `main` branch with `git branch duplicate main` (the duplicate branch will be named `duplicate`);
- checkout your main branch with `git checkout main` (in case you have not already checked out it);
- hard reset you main branch to the commit that preceds `af41f05cfc` with `git reset --hard af41f05cfc`;
- cherry-pick the commit for this pull request with `git cherry-pick af41f05cfc`;
- push the change with `git push`;
- put back d4f7257766 in your main branch with `git cherry-pick d4f7257766`.

In general, I suggest you use a new branch for each change and use your main branch to track upstream only. Once a change is ready in some branch, then you cherry-pick it in your main branch and push. (For changes that need more commits in the same branch, you would need to play with `git rebase` and `git merge`, but I guess this is not needed much in doc.)

You should be safe experimenting with git as long as you ensure the two following conditions are always satisfied:
- whenever you use `git reset` or some other command that drops commits you still have the same commits in another backup branch;
- whenever you `git push` you check carefully what you are actually pushing.

**Comment by grahamperrin** [comment]
> … I'll not push to `freebsd main` until after the 13.2-related doc slush ends. …

Through trying to work with branches with `git` at the command line, I ended up with 

* two commits pushed too soon
* the one commit that I _did_ want pushed, not pushed. 

For now, I prefer to not discuss command-line use.

**Comment by lsalvadore** [comment]
Thanks @grahamperrin for the commit and if you ever want want or need to train your git skills feel free to ask me for help if you want. You might also want to consider creating a new git repository completely unrelated to FreeBSD and fully in your control to practice with extra safety.

**Pull request closed without merging**